### PR TITLE
Add documentation refs for argmax and argmin

### PR DESCRIPTION
**Pull request by Seelengrab**
These were missing on some methods, so I added them.

Doesn't really need to be backported to 1.7, would be a nice to have though.

[no-ci]

### CODE REVIEW

**Comment by Seelengrab** [comment]
Seems like my no-ci incantation didn't work - does this feature even exist in this repo?

**Comment by simeonschaub** [comment]
It's `[ci skip]` (or equivalently `[skip ci]`). We still want to run CI on docstring changes like this though to check whether the references are actually valid.

**Comment by Seelengrab** [comment]
Ah, I see! That makes sense.

Is there maybe an incantation to only run CI for documentation? Seems like the doctest CI is it's own build task, would cut down on CI time for doc-PRs.

**Comment by simeonschaub** [comment]
No, not currently.

**Comment by Seelengrab** [comment]
doctests run through so I think the refs are fine - not sure what's happening with the package builds 🤷‍♂️

File: base/reduce.jl [hunk 1]
```diff
@@ -900,6 +900,8 @@ Values are compared with `isless`.
 !!! compat "Julia 1.7"
     This method requires Julia 1.7 or later.
 
+See also: [`argmin`](@ref), [`findmax`](@ref).
```
Comment (fredrikekre) [inline base/reduce.jl:903]:
```suggestion
See also [`argmin`](@ref), [`findmax`](@ref).
```

File: base/reduce.jl [hunk 2]
```diff
@@ -950,6 +952,8 @@ If there are multiple minimal values for `f(x)` then the first one will be found
 !!! compat "Julia 1.7"
     This method requires Julia 1.7 or later.
 
+See also: [`argmax`](@ref), [`findmin`](@ref).
```
Comment (fredrikekre) [inline base/reduce.jl:955]:
```suggestion
See also [`argmax`](@ref), [`findmin`](@ref).
```

**Review by fredrikekre — APPROVED** [review]
https://github.com/JuliaLang/julia/pull/40401

File: base/reduce.jl (hunk 1 above)
Comment (Seelengrab) [reply base/reduce.jl:903]: Seems like that PR was not consistently applied across Base, I've just followed the convention in that file:

```
julia $ rg "See also\s" base/ | wc -l          
230                                                            
julia $ rg "See also:\s" base/ | wc -l         
131                                                            
julia $ rg "See also\s" base/reduce.jl | wc -l 
0                                                              
julia $ rg "See also:\s" base/reduce.jl | wc -l
11                                                             
```

Should I change it in all of reduce.jl? There are some erronous `,` in other docstrings as well, looking through them.

Comment (fredrikekre) [reply base/reduce.jl:903]: Hmm, maybe those were merged after that PR then. Could do a follow up PR to fix the rest, but no need to introduce more of them.

Comment (Seelengrab) [reply base/reduce.jl]: I don't know, it just seems like a lot of `See also`s - is there some hook in Documenter.jl or some doc CI where such things could be checked?

Comment (fredrikekre) [reply base/reduce.jl]: No it is just a convention (as documented in the manual).

**Pull request merged**